src/richie/apps/core/templates/richie/fullwidth.html

```diff
@@ -5,6 +5,12 @@
     {% page_attribute "page_title" %}
 {% endblock title %}
 
+{% block meta_opengraph_contextuals %}
+    {% page_attribute "page_title" as page_title %}
+    <meta property="og:title" content="{{ page_title|truncatechars:65 }}">
+    <meta property="og:url" content="{{ SITE.web_url }}{{ current_page.get_absolute_url }}">
+{% endblock meta_opengraph_contextuals %}
```

Comment:
shouldn't you put the logo as image in the meantime that we figure out how to retrieve the page's image?

Reply:
For now, every page will inherit from the site logo for the og:image, until we implement a way to get image. But note than templates like fullwidth or single_column only have a single main placeholder for everything, we won't be able to find any relevant image so they probably always inherit from site logo.

Reply:
yes and no... you could for example look for the first plugin image linked to that placeholder...

Reply:
or add a cover plugin that would enable them for use as a plugin...